### PR TITLE
Login and create account form styling

### LINKED ISSUE #78: "No Jobs Completed"
If a Job Lister has not completed any jobs themselves (as a contractor) then their public profile should show something like "No Jobs Completed" on the "My Jobs" tab

### PR DESCRIPTION
**Pull request by ToniRib**
closes #78 

Did a bunch of random little things that needed to be done including:
- validations for the bio
- validations for the email address
- adding a "create account" link to the log in page
- styling for create account form and adding "required: true" to the fields
- friendly forwarding for the job show page, so if you are a guest and click "log in to place a bid" and then you log in, you are brought back to the job page instead of the dashboard
- other random style updates

### CODE REVIEW

**Comment by ToniRib** [comment]
@jwperry @danjwinter this pull request is ready for you to review!

File: test/integration/guest_views_job_show_page_test.rb [hunk 1]
```diff
@@ -17,6 +17,22 @@ class GuestViewsJobShowPageTest < ActionDispatch::IntegrationTest
     verify_lister_info(lister)
   end
 
+  test "guest views job show page while bidding is open - friendly forwarding" do
```
Comment (ToniRib) [inline test/integration/guest_views_job_show_page_test.rb:20]: I'm going to ignore this. It's only too long by one character.

Comment (jwperry) [inline test/integration/guest_views_job_show_page_test.rb:20]: Not an issue. Back off, Hound!

**Comment by jwperry** [comment]
Looks great!

**Pull request merged**